build.gradle

```diff
@@ -38,12 +38,14 @@ checkstyle {
     getConfigDirectory().set(new File(rootDir, 'config/checkstyle'))
 }
 
-def sonarExclusions = ['**/uk/gov/hmcts/ethos/replacement/docmosis/config/**',
-                       '**/uk/gov/hmcts/ethos/replacement/docmosis/model/**',
-                       '**/uk/gov/hmcts/reform/ethos/replacement/docmosis/CaseOrchestrationApplication',
-                       '**/uk/gov/hmcts/ethos/replacement/docmosis.config.DevQueueClient.java',
-                       '**/uk/gov/hmcts/ethos/replacement/docmosis/domain/admin/**',
-                       '**/uk/gov/hmcts/ethos/replacement/docmosis/controllers/RespondentTellSomethingElseController.java'
+def sonarExclusions =
+        ['**/uk/gov/hmcts/ethos/replacement/docmosis/config/**',
+         '**/uk/gov/hmcts/ethos/replacement/docmosis/model/**',
+         '**/uk/gov/hmcts/reform/ethos/replacement/docmosis/CaseOrchestrationApplication',
+         '**/uk/gov/hmcts/ethos/replacement/docmosis.config.DevQueueClient.java',
+         '**/uk/gov/hmcts/ethos/replacement/docmosis/domain/admin/**',
+         '**/uk/gov/hmcts/ethos/replacement/docmosis/controllers/RespondentTellSomethingElseController.java',
+         '**/uk/gov/hmcts/ethos/replacement/docmosis/controllers/CaseActionsForCaseWorkerController.java'
```

Comment:
can we revert the exclusion change? shouldn't be excluding files from sonar without reason

Reply:
done